src/DetailsView/components/switcher.scss

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../common/styles/colors.scss';
+
+// The two layers of class selector and the explicit enumeration of the states are both
+// required to beat the specificity of office fabric styles
+.header-switcher .header-switcher-dropdown {
+    &,
+    &:focus,
+    &:hover,
+    &:active,
+    &:hover:active {
+        :global(.ms-Dropdown-title) {
+            background-color: $ada-brand-variant;
+            height: 32px;
+            border: 2px $switcher-border solid;
+            box-sizing: border-box;
+            border-radius: 2px;
+            width: 165px;
+            color: $neutral-0;
+            i {
+                margin-right: 8px;
+            }
+        }
+        :global(.ms-Dropdown-caretDownWrapper) {
+            top: 4px;
+        }
+        :global(.ms-Dropdown-caretDown) {
+            color: $neutral-0;
+        }
+    }
+
+    &:focus {
+        .ms-Dropdown-title {
```

Comment:
does `.ms-Dropdown-title` here needs to be wrapped into `global` just like in line 13?

Reply:
Yes, good catch